docs/association/legal/Privacy_policy.md

```diff
@@ -1,63 +1,90 @@
 
-Aragon processes personal data to the extent that at least one of the following applies:
+**For suppliers receiving funds from Aragon Association**  
+The user acknowledges the use of the data included in Supplier Registration Form to be processed by the Aragon Association within the scope of the commercial relation existing between the Parties. 
 
-1. The Data Subject has given consent for one or more specific purposes. 
+The user consents that the information included in the Supplier Registration Form can be disclosed at any time by the Aragon Association as open information accessible by the Aragon Community and according to Aragon transparency principles and democartic values, including any personal data and bank account, services prices, methods of payments or other financial information regarding the services provided to the Aragon Association. 
 
-1. Processing is necessary for the performance of our agreements with the Data Subjects. 
+The user consent does not preclude the right to exercise any data privacy rights as data subject according to the applicable data privacy laws and regulations and Aragon’s Privacy Policy. 
 
-1. Processing is necessary to comply with our legal obligations.
+The user is aware that the payment for the services provided may be executed throughout the blockchain and acknowledges that the information included in the Supplier Registration Form will be registered and public, without any technical possibility of reversing this process or eliminating the data, if no special request for anonymization or exclusion of the data is made previously. 
 
+## 4. How the information is shared
 
-## Disclosure to other recipients
+Aragon shares personal information with other Aragon companies, among the Aragon group or third party service providers that assist Aragon providing services. This may include sharing information with third party contractors, bound by obligations of confidentiality, in connection with the processing of your personal information for the purposes described in this Policy, such as, but not limited to, IT and communications service providers, third parties relevant to the services that Aragon provides including regulators, authorities and governmental institutions.
+	
+Aragon may transfer personal information outside Europe. In those cases Aragon will ensure that it is protected and transferred in a manner consistent with legal requirements applicable to the information.
 
-Aragon does not share the personal data of the Data Subjects with any third party, other than to Aragon providers to the extent necessary to render our services or to competent authorities if required under a mandatory regulation or requirement.
+When the data subject expressed consent Data will be shared between Aragon Association, Aragon One and other partners of the Aragon Network that require the data in order to adequately perform their duties. 
```

Comment:
(There is no need for paragraph 65. We already say above how we share data - p. 61.